README-LICENSE.md

```diff
@@ -0,0 +1,63 @@
+
+# Licensing
+
+Unless noted below, NeoForge and its all parts here in this repository are
```

Comment:
```suggestion
Unless noted below, NeoForge and all its parts here in this repository are
```

Reply:
Whoops. That's a holdover from before I removed "NeoForge Mod Loader" and inserted "its" (from the original, "[...] NeoForge, NeoForge Mod Loader, and all parts herein are licensed [...]").